plugin/src/main/kotlin/tanvd/grazi/GraziConfig.kt

```diff
@@ -17,10 +17,23 @@ class GraziConfig : PersistentStateComponent<GraziConfig.State> {
                      @Property val userEnabledRules: Set<String> = HashSet(),
                      @Property val lastSeenVersion: String? = null) {
 
+        val availableAndEnabledLanguages: Set<Lang>
+            get() = enabledLanguages.filter { it.jLanguage != null }.toSet()
+
         fun clone() = State(
                 enabledLanguages = HashSet(enabledLanguages), nativeLanguage = nativeLanguage, enabledSpellcheck = enabledSpellcheck,
                 userWords = HashSet(userWords), userDisabledRules = HashSet(userDisabledRules), userEnabledRules = HashSet(userEnabledRules),
                 lastSeenVersion = lastSeenVersion)
+
+        fun hasMissedLanguages() = nativeLanguage.jLanguage == null || hasMissedLanguagesWithoutNative()
+
+        fun hasMissedLanguagesWithoutNative() = enabledLanguages.any { it.jLanguage == null }
```

Comment:
just add param in function above `withNative: Boolean = false`

Reply:
Done